pkg/policy/k8s/network_policy.go

```diff
@@ -31,10 +31,6 @@ func (p *policyWatcher) addK8sNetworkPolicyV1(k8sNP *slim_networkingv1.NetworkPo
 	}
 	scopedLog = scopedLog.WithField(logfields.K8sNetworkPolicyName, k8sNP.ObjectMeta.Name)
 
-	if k8s.NetworkPolicyHasEndPort(k8sNP) {
-		scopedLog.Warning("EndPort in kubernetes NetworkPolicy is not supported")
-	}
```

Comment:
Awesome! Is there an e2e test we can now enable?

Reply:
I couldn't find anything. If you come across something, please let me know. The [Kubernetes test](https://github.com/cilium/cilium/blob/main/test/kubernetes-test.sh) does not have exclusions for port ranges.

Reply:
:raised_eyebrow: There must be tests somewhere, it's [stable in k8s since late 2022](https://kubernetes.io/blog/2022/08/23/kubernetes-v1-25-release/#promoted-endport-in-network-policy-to-stable). We shouldn't be merging functionality like this without a plan for automated testing.